spec/logr/entry_spec.rb

```diff
@@ -0,0 +1,118 @@
+require 'spec_helper'
+
+describe Logr::Entry do
+  let(:logger) { double('logger') }
+  let(:entry) { Logr::Entry.new(logger) }
+
+  describe '#event' do
+    let(:entry_event) { entry.event('test_event', context_1_key: 'context_1_value') }
+
+    it 'should add the event with the correct name' do
+      expect(entry_event.instance_variable_get(:@event).name).to eq('test_event')
+    end
+
+    it 'should add the event with the correct context' do
+      expect(entry_event.instance_variable_get(:@event).context).to include(context_1_key: 'context_1_value')
```

Comment:
I'm only using instance_variable_get, because I couldn't create a proper attr_reader. The name clashes with the `event` method. But I'm open for suggestions on how to better test this. My problem with the current method is that this tests both Event and Entry and not just Entry as it should.